chain/network/src/peer/peer_actor.rs

```diff
@@ -340,6 +340,9 @@ impl PeerActor {
     }
 
     fn receive_view_client_message(&self, ctx: &mut Context<PeerActor>, msg: PeerMessage) {
+        metrics::PEER_VIEW_CLIENT_MESSAGE_RECEIVED_BY_TYPE_TOTAL
+            .with_label_values(&[msg.msg_variant()])
+            .inc();
```

Comment:
Or if we want simple separation to client and view client messages have ‘near_peer_client_message_received_count{category, type}’ with ‘client’ and ‘view_client’ categories.

Reply:
Moved to receive_message and get rid of the redundant metric `PEER_CLIENT_MESSAGE_RECEIVED_TOTAL`. I'm still keeping these two metrics separate because I want to show them in separate grafana graphs.

Reply:
That’s what I meant by my second comment. We could easily have two labels. It is usually easier to deal with fewer variables with labels rather than with multiple variables.